Automattic/jetpack
# Add ProductPrice discount label component

#### Description
This PR introduces a discount label element for shared usage within the ProductPrice component for the unification of the interstitial pricing pages for standalone Jetpack plugins.

#### Other information:

#### Jetpack product discussion


- Asana
- P2

#### Does this pull request change what data or activity we track or use?


- No

#### Testing instructions:





- Checkout this branch
- Add a `hideDiscountLabel` parameter to an active ProductPrice component and set it to `false`
- Start your Jurassic Tube and build the plugin
- View the pricing page and verify that the discount is found and displays/functions as expected
